config.yaml

```diff
@@ -0,0 +1,27 @@
+document: modularity-testing
```

Comment:
Hi Merlin,
This file defines some important values for modules, for example location of docker images, or rpm repos, source, where is modulemd file.
Yep, some of them may be defined on CMD line as you mentioned URL, to refedine these.
It means, that in case I'll not pass URL and some options, these options will be fetch from that file. from that perspective it could be takem as default values for various module types. 
For baseruntime this file probably could be default. just changed links to proper places with modulemd file, and repo, and docker.io registry. But in other cases for you this file is not so important because baseruntime does not have public facing (like running some services ad port forwarding to host machine)
because in these cases I'll pass there way how to start this module.
I can show it on memcached module: docker should be started as `docker run -it -e CACHE_SIZE=128   -p 11211:11211` and for example rpm based as `systemctl start memcached` and actually there is no machine readable format for that, so I had to define that somehow on one place (for various types of module)